src/rgw/rgw_admin.cc

```diff
@@ -1,1011 +1,55 @@
 // -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
 // vim: ts=8 sw=2 smarttab
 
-#include <errno.h>
+#include <cerrno>
 #include <iostream>
 #include <sstream>
+#include <cstdlib>
 #include <string>
 
 #include <boost/optional.hpp>
+#include <unordered_set>
+#include <map>
 
 #include "auth/Crypto.h"
 #include "compressor/Compressor.h"
 
-#include "common/armor.h"
 #include "common/ceph_json.h"
 #include "common/config.h"
-#include "common/ceph_argparse.h"
-#include "common/Formatter.h"
-#include "common/errno.h"
-#include "common/safe_io.h"
 
```

Comment:
are you sure you don't need common/errno.h?

Reply:
I fixed #includes and now newly introduced .h files don't include common/errno.h, so I think I don't.